packages/core/useCloned/index.md

```diff
@@ -0,0 +1,60 @@
+---
+category: Utilities
+---
+
+# useCloned
+
+Reactive partial clone.
+
+## Usage
+```ts
+import { useCloned } from '@vueuse/core/useCloned'
+
+const originData = ref({
+  key: 'value'
+})
+
+const { cloned } = useCloned(originData)
+
+originData.key = 'some new value'
+
+console.log(cloned.value.key) // 'some new value'
+
+```
+
+## Manual cloning
+```ts
+import { useCloned } from '@vueuse/core/useCloned'
+
+const data = ref({
+  key: 'value'
+})
+
+const { cloned, sync } = useCloned(data, { manual: true })
+
+data.key = 'manual'
+
+console.log(cloned.value.key) // 'value'
+
+sync()
+
+console.log(cloned.value.key)// 'manual'
+```
+
+## Custom clone function usage
+```ts
+import { useCloned } from '@vueuse/core/useCloned'
+
+const data = ref({
+  key: 'value'
+})
+
+const { cloned, sync } = useCloned(data, {
+  cloneFunction: (source, cloned) => ({ ...source, isCloned: true })
```

Comment:
We should show a case of using `deepclone` or something instead of show the example for adding extra properties (which kinda violate the idea of `cloneing`)